dom.go

```diff
@@ -69,6 +69,13 @@ type Unmounter interface {
 	Unmount()
 }
 
+// Keyer is an optional interface that a Component can implement in order to
+// uniquely identify the component amongst its siblings. If implemented, all
+// sibling components and HTML must also be keyed.
```

Comment:
As it stands:

> If implemented, all sibling components and HTML must also be keyed.

Makes it sounds like "all sibling components" *and* "HTML" must be keyed, so maybe even children of children? I think that's not the intent, so better wording would be:

> If implemented, all siblings, both component and HTML, must also be keyed.